tests/integration/test_authd/test_authd_worker.py

```diff
@@ -8,7 +8,7 @@
 
 import pytest
 import yaml
-from wazuh_testing.cluster import FERNET_KEY, CLUSTER_DATA_HEADER_SIZE, cluster_msg_build
+from wazuh_testing.tools.key_polling_cluster import FERNET_KEY, CLUSTER_DATA_HEADER_SIZE, cluster_msg_build
```

Comment:
```suggestion
from wazuh_testing.cluster import FERNET_KEY, CLUSTER_DATA_HEADER_SIZE, cluster_msg_build
```
Fix the usage of `cluster_msg_build` from below rather than changing the import.